war/src/main/js/add-item.js

```diff
@@ -1,6 +1,6 @@
 // Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
 var $ = require('jquery-detached').getJQuery();
-var $remodal = require('remodal-detached').getRemodal();
+require('remodal-detached').getRemodal();
```

Comment:
This is like a bootstrap library. I do not need to use the rules by myself. But I need predifined rules for the standard behaviour